com.aqtiongame.AQtion.yml

```diff
@@ -0,0 +1,160 @@
+  - /app/bin/patchelf
+  - "*.a"
+  - "*.la"
+
+modules:
+  - name: libjpeg # with libjpeg.so.8
+    buildsystem: cmake-ninja
+    config-opts: &libjpeg_config_opts
+      - -DCMAKE_SKIP_RPATH:BOOL=YES
+      - -DENABLE_STATIC:BOOL=NO
+      - -DWITH_JPEG8:BOOL=YES
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib # uses lib64 by default
+    sources: &libjpeg_sources
+      - type: archive
+        url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/refs/tags/2.1.3.tar.gz
+        sha256: dbda0c685942aa3ea908496592491e5ec8160d2cf1ec9d5fd5470e50768e7859
+
+  - name: libpng
+    buildsystem: cmake-ninja
+    sources: &libpng_sources
+      - type: archive
+        url: https://github.com/glennrp/libpng/archive/v1.6.35.tar.gz
+        sha256: 6d59d6a154ccbb772ec11772cb8f8beb0d382b61e7ccc62435bf7311c9f4b210
+
+  - name: patchelf
+    buildsystem: simple
+    sources: &patchelf_sources
+      - type: archive
+        url: https://github.com/NixOS/patchelf/archive/refs/tags/0.18.0.tar.gz
+        sha256: 1451d01ee3a21100340aed867d0b799f46f0b1749680028d38c3f5d0128fb8a7
+    build-commands:
+      - ./bootstrap.sh
+      - ./configure --prefix=/app
+      - make
+      - make install
+
+  - name: rsync
+    config-opts:
+      - --prefix=${FLATPAK_DEST}
+      - --with-included-popt
+      - --with-included-zlib
+      - --host=x86
+      - --disable-xxhash
+      - --disable-md2man
+    sources:
+      - type: archive
+        url: https://download.samba.org/pub/rsync/src/rsync-3.2.7.tar.gz
+        sha256: 4e7d9d3f6ed10878c58c5fb724a67dacf4b6aac7340b13e488fb2dc41346f2bb
+        x-checker-data:
+          type: anitya
+          project-id: 4217
+          stable-only: true
+          url-template: https://download.samba.org/pub/rsync/src/rsync-$version.tar.gz
+    cleanup:
+      - /share/man
+
+  - name: discord_sdk
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/actionquake/distrib/raw/main/build/linux/integrations/discord_game_sdk.so
+        sha256: f52f9a7f9358f07a18e65a9e26a2943caf73cd681b2fea015c95a32e4239eb17
+    build-commands:
+      - mv discord_game_sdk.so /app/lib/discord_game_sdk.so
+
+  - name: q2pro  # Credit to fishxz - https://github.com/flathub/com.github.skullernet.q2pro
+    buildsystem: meson
+    builddir: true
+    build-options:
+      config-opts:
+        - --auto-features=enabled
+        - --fatal-meson-warnings
+        - -Danticheat-server=true
+        - -Dclient-gtv=true
+        - -Dpacketdup-hack=true
+        - -Dtests=false
+        - -Dvariable-fps=true
+        - -Dwerror=false
+        - -Daqtion-build=true
+        - -Ddiscord-sdk=true
+        - -Ddatadir=~/.var/app/com.aqtiongame.aqtion/data
+        - -Dhomedir=~/.var/app/com.aqtiongame.aqtion/data
+        - -Dwindows-crash-dumps=disabled
+        - -Dwayland=disabled
+        - -Dwrap_mode=nofallback
+    post-install:
+      - install -Dm644 ../256_aqtion.png -t ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps
```

Comment:
since the icons don't come from upstream, there should be only ONE size: the biggest. Or SVG.